src/pages/sns/faq.tsx

```diff
@@ -741,6 +808,44 @@ function SnsFaqPage() {
                 </Link>
               </p>
             </Faq>
+            
+            <Faq title="What is voting power and how do I get it?">
+              SNS DAOs have a stake-based governance system. This means the more
```

Comment:
I am missing the full context of the page, but is it clear what a "neuron" is to the reader or can they easily find this infomration?